### ISSUE TITLE
Feature request: assess contrast of link text relative to non-link text.

### ISSUE THREAD

**Issue by neil-h** [Khan/tota11y#59]
tota11y is flagging links on a colored background for having insufficient contrast, but if I change the colors to its recommendation, I find, even with my exceptional eyesight, I have difficulty distinguishing the links from the regular text.

It would be great if there was some tool to assess the contrast of link text. I've been going back and forth trying to make the links have sufficient contrast from both the background, and the other text.

**Comment by jdan** [comment]
Hey @neil-h, sorry for not getting back to you sooner. This is a really good point, have you considered a different type of visual indicator for links besides having a different color from normal text? Typically, links should be underlined.

**Comment by neil-h** [comment]
I know this used to be standard, but it doesn't seem to be anymore. Wikipedia doesn't underline links, unless you hover them. That's also the default for Bootstrap. It looks like Zurb Foundation doesn't even do that.

**Comment by jdan** [comment]
@neil-h Good point :) I'll look into this - hopefully someone more knowledgeable can chime in here.

**Comment by jdan** [comment]
As pointed out by @dsturley in the web-a11y slack room, there are such guidelines!

http://www.w3.org/TR/2015/NOTE-WCAG20-TECHS-20150226/G183

> With this technique, a relative luminance (lightness) difference of 3:1 or greater with the text around it can be used if additional visual confirmation is available when a user points or tabs to the link. Visual highlights may, for example, take the form of underline, a change in font style such as bold or italics, or an increase in font size.
